src/Responses/Json10/Properties/ItemVariant.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Json10\Properties;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+
+class ItemVariant extends BaseItem
```

Comment:
This empty class exists to:
- Make type hinting more clear
- Make sure we can easily expand the variant class, without breaking anything for existing usages